nodebuilder/share/service.go

```diff
@@ -27,9 +27,14 @@ import (
 //
 // Any method signature changed here needs to also be changed in the API struct.
 type Module interface {
-	share.Availability
+	// SharesAvailable subjectively validates if Shares committed to the given Root are available on the Network.
+	SharesAvailable(context.Context, *share.Root) error
+	// ProbabilityOfAvailability calculates the probability of the data square
+	// being available based on the number of samples collected.
+	ProbabilityOfAvailability() float64
```

Comment:
Feel conflicted about including this method in the API, as it's inherently broken. You can ignore this comment

Reply:
@renaynay, WDYT?

Reply:
Hmm @Wondertan I think it's a nice to have (when it's properly implemented -- mean calculated based on num samples), but this is totally post mainnet so I think we can remove the method for now in general. I don't see the use in the way it's calculated rn other than providing users ability to calculate probability for light avail given default sample size.